darwin/importer/importer.py

```diff
@@ -491,6 +501,12 @@ def import_annotations(
                     import_annotators,
                     import_reviewers,
                 )
+                if potential_error:
+                    errors.append((parsed_file.full_path, potential_error))
+        if errors:
+            console.print(f"Encountered errors on {len(errors)} imports", style="error")
```

Comment:
This isn't your doing, but we need to improve our handling of `Console` objects, as they are only present for CLI code, not SDK.  The dataset carries a `console` property that allows us to tell whether this is an intereactive session or not, and we should probably move to using that at top end of the function.

Not something for now unless you want to do it.